Assets/Scripts/FlatMapInteraction.cs

```diff
@@ -158,7 +158,7 @@ public void OnPointerClick(PointerEventData eventData)
 
         map.UpdateMap(geo, map.Zoom);
         ZoomIn();
-        statemanager.UpdateSites(false);
+        statemanager.UpdateEntities(false);
         // TODO: Fix RescaleSiteMarkers to work after UpdateSites
         // Currently broken because of async issues.
         //RescaleSiteMarkers();
```

Comment:
Since UpdateEntities updates wans as well the line below this calling UpdateWans is redundant so you can remove it.

Reply:
Removed: 45c655c